### PR TITLE
"Populated places" from "Towns" and "Settlements" layers

### PR DESCRIPTION
**Pull request by trey-stafford**
## Description

Combine the "Places/Towns" and "Places/Settlements" layers into one "Places/Populated places" layer. Update style to change marker-size based on population size.

In QGreenland v2, a similar layer with a `population` attribute was  "Places/Community map (crowd-sourced)/Populated places" layer from [HOTOSM](https://www.hotosm.org/). This layer gets used in some of our tutorial materials, including our [QGreenland Beginner's Tutorial Series, Session 5: Processing Toolbox and Data Analysis YouTube video](https://www.youtube.com/watch?v=znKeiV3-Amo&list=PLSRiyMridUCwyu-vqpAFtm8bVERgTvs7q&index=6). 

We removed all of the HOTOSM layers due to feedback indicating numerous inaccuracies. See https://github.com/nsidc/qgreenland/pull/540. 

This PR provides a suitable replacement layer for our tutorial materials. Although this layer will be in the root of the "Places" group instead of the  "Community map (crowd-sourced)" subgroup (as indicated in the YouTube tutorial), the user experience of selecting a layer called "Populated places" in **Processing Toolbox** tools will be the same.

A couple of things to consider to make this dataset more consistent with the original "Populated places" layer:

* Rename the "Population  2016" field to "population"? This would be consistent with the original layer, but we lose information about when the population data is valid for. This could be included in the layer description, but I lean toward keeping it as-is ("Population 2016"). Another option would be to duplicate the column as "population", but that could also lead to confusion. 
* Filter out places with a population of 0. This would address most of our "Noted data issues" in the layer description. Also, these places are by definition without a population, so having them be a part of a layer called "Populated places" may not make sense anyway. Is there value in keeping these records? Putting them into a different layer? 

## Checklist

If an item on this list is done _or not needed_, simply check it with `[x]`.

- [x] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [x] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build`)
- [x] CHANGELOG.md updated
- [x] Documentation updated if needed
- [x] New unit tests if needed

### CODE REVIEW

**Comment by MattF-NSIDC** [comment]
I suppose that's a huge downside of tutorials on YouTube. By their in-depth nature, tutorials are easy to break with minor changes...

Maybe our practice should be to update the descriptions of those videos with a full list of changes? Perhaps we could also edit the video to add an initial splash screen that explains that the tutorial needs adjustments (found in video description) to work with new versions of QGreenland.

File: doc/user/tutorials/interacting-with-geospatial-data.md [hunk 1]
```diff
@@ -241,14 +240,15 @@ What is the average size of Greenland’s populated areas?
...
 
-The **Vector Analysis** window should automatically switch to a view of the Log where you will
-see the results for the population basic statistics (Fig. 8). The value ‘MEAN’ will tell you the
-average size of Greenland’s populated places (1102 people). The value ‘SUM’ will tell you the
-total number of people in Greenland’s metropolitan areas (48,492 people).
+The **Basic statistics for fields** window should automatically switch to a view
+of the Log where you will see the results for the population basic statistics
+(Fig. 8). The value ‘MEAN’ will tell you the average size of Greenland’s
+populated places (702 people). The value ‘SUM’ will tell you the total number
+of people in Greenland’s metropolitan areas (55,494 people).
```
Comment (MattF-NSIDC) [inline doc/user/tutorials/interacting-with-geospatial-data.md:251]: Nice!

File: doc/user/tutorials/video-series-overview.md [hunk 2]
```diff
@@ -58,23 +58,12 @@ If you are not familiar with geographic information systems (GIS) and geospatial
...
-1. Open up the attribute table, and click the ‘Open field calculator’ icon.
-
-    ![OpenFieldcalculator_Larger](/_images/OpenFieldcalculator_Larger.png)
-
-2. Check ‘Create a new field’. Designate an output field name (example: pop_numeric) and
-   select the output field type ‘Integer (32 bit)’. In the expression panel, choose the
-   field ‘population’. Click OK.
-
-    ![create_new_field](/_images/create_new_field.png)
+This updated "Populated places" layer has a "Population 2016" attribute that should be used instead of the "population" attribute indicated in the video.
```
Comment (MattF-NSIDC) [inline doc/user/tutorials/video-series-overview.md:64]: Is this updated in the YouTube video description too?

File: qgreenland/config/datasets/asiaq_nunagis.py [hunk 3]
```diff
@@ -15,15 +16,19 @@
...
+            args=[
+                "wget",
+                "-O",
+                "{output_dir}/settlements.geojson",
+                '"https://kort.nunagis.gl/refserver/rest/services/Kortportal/Byer_og_bygder/MapServer/0/query/?f=json&where=OBJECTID+is+not+null&outFields=*&orderByFields=OBJECTID+ASC"',
+                "&&",
+                "wget",
+                "-O",
+                "{output_dir}/towns.geojson",
+                '"https://kort.nunagis.gl/refserver/rest/services/Kortportal/Byer_og_bygder/MapServer/1/query/?f=json&where=OBJECTID+is+not+null&outFields=*&orderByFields=OBJECTID+ASC"',
```
Comment (MattF-NSIDC) [inline qgreenland/config/datasets/asiaq_nunagis.py:30]: Simple and clean solution, nice :)

**Review by MattF-NSIDC — APPROVED** [review]

File: doc/user/tutorials/video-series-overview.md (hunk 2 above)
Comment (trey-stafford) [reply doc/user/tutorials/video-series-overview.md:64]: Just updated!

**Pull request merged**